fastly/fixtures/versions/get.yaml

```diff
@@ -9,24 +9,30 @@ interactions:
       Fastly-Key:
       - xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
       User-Agent:
-      - FastlyGo/0.2 (+github.com/sethvargo/go-fastly; go1.8.3)
-    url: https://api.fastly.com/service/7i6HN3TK9wS159v2gPAZ8A/version/696
+      - FastlyGo/0.4.3.dev (+github.com/sethvargo/go-fastly; go1.12)
+    url: https://api.fastly.com/service/7i6HN3TK9wS159v2gPAZ8A/version/16
     method: GET
   response:
-    body: '{"testing":false,"locked":false,"number":696,"active":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","staging":false,"created_at":"2017-07-20T01:15:13+00:00","deleted_at":null,"comment":"","updated_at":"2017-07-20T01:15:13+00:00","deployed":false}'
+    body: '{"testing":false,"locked":false,"number":16,"active":false,"service_id":"7i6HN3TK9wS159v2gPAZ8A","staging":false,"created_at":"2019-03-29T00:33:36Z","deleted_at":null,"comment":"test
+      comment","updated_at":"2019-03-29T00:33:36Z","deployed":false}'
     headers:
       Accept-Ranges:
       - bytes
+      - bytes
+      - bytes
```

Comment:
This is probably a bug in the VCR recorder, but this is leaking. No action required on your part, more of a mental note for me.

Reply:
This seems like a bug. I have checked my latest PRs and they all have the same issue with `Accept-Ranges` in the recordings.